app/src/main/res/menu/feeditemlist_context.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:id="@+id/episode_actions"
```

Comment:
Please call it `multi_select` instead. `episode_actions` does not tell readers that it is about selecting multiple items, not about actions for a single item.